tests/backup_helper.go

```diff
@@ -4465,6 +4465,8 @@ func PxBackupUpgrade(versionToUpgrade string) error {
 	log.Infof("Upgrading Px-Backup version from %s to %s", currentBackupVersionString, versionToUpgrade)
 	customRegistry := os.Getenv("CUSTOM_REGISTRY")
 	customRepo := os.Getenv("CUSTOM_REPO")
+	customRegistry = "docker.io"
```

Comment:
Why do we need this? Ideally we can pass it from the job right?

Reply:
Added just for testing purpose because the BYOC pipeline did not have this parameter configured